environments/sandbox.yml

```diff
@@ -264,7 +264,11 @@ cname:
     record: "sdshmcts-sbox.azurefd.net"
   - name: "labs-rhodrif-nodejs"
     ttl: 300
+    shutter: false
```

Comment:
no i meant this goes in azure-platform-terraform so it doesn't create a shutter app